compatibility/history-proxy/js/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "history-proxy",
+  "version": "0.0.1",
+  "main": "index.js",
+  "author": {
+    "name": "Francisco Angelo Cabelo",
+    "email": "francisco.a.cabelo@gmail.com",
+    "url": "https://github.com/cfrancisco"
+  },
+  "description": "The GUI Proxy allows the Dojot GUI v1 to be compliant with InfluxDb, which is used in newer versions of Dojot.",
+  "scripts": {
+    "lint": "eslint src/**",
+    "start": "node index.js",
+    "dev": "node --trace-sync-io --unhandled-rejections=strict index",
+    "test": "jest --verbose",
+    "check-licenses": "apache2-license-checker"
+  },
+  "jest": {
+    "coverageDirectory": "./coverage/",
+    "collectCoverage": true,
+    "testEnvironment": "node",
+    "coveragePathIgnorePatterns": [
+      "/node_modules/"
+    ]
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/dojot/dojot"
+  },
+  "license": "Apache-2.0",
+  "licenses": [
+    {
+      "type": "Apache-2.0",
+      "url": "http://www.apache.org/licenses/LICENSE-2.0"
+    }
+  ],
+  "keywords": [
+    "proxy",
+    "dojot",
+    "gui"
+  ],
+  "private": false,
+  "devDependencies": {
+    "@types/jest": "^26.0.10",
+    "apache2-license-checker": "^1.0.0",
+    "eslint": "^7.19.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.21.1",
+    "eslint-plugin-jest": "^24.1.5",
+    "eslint-plugin-security": "^1.4.0",
+    "jest": "^26.6.3",
+    "mocha": "^8.3.0",
+    "prettier": "^2.1.1",
+    "superagent-defaults": "^0.1.14",
+    "supertest": "^6.1.3"
+  },
+  "dependencies": {
+    "@dojot/microservice-sdk": "0.1.9",
+    "axios": "^0.21.1",
+    "express": "^4.17.1",
+    "express-async-errors": "^3.1.1",
+    "express-http-proxy": "^1.6.2",
```

Comment:
I think the `express-http-proxy` and http-proxy-middleware` were not used. 
Check for other cases....

Reply:
Thanks! You are right, I just removed it. I also checked the other dependencies.